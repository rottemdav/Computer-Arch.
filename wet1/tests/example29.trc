32 5 25 2 global_history global_tables not_using_share
0x6010 T 0xa0c38
0xa0404cc N 0x53010350
0xa00 T 0x9450165c
0x4f80370 T 0xf1d54
0xa0404cc T 0x53010350
0xa0404cc N 0x53010350
0x1b208 N 0x3394
0x0b8 N 0x7f38
0xa00 N 0x9450165c
0xa0b14 N 0xbdc8930
0xcb8 T 0xb529f5e4
0x6010 N 0xa0c38
0x796f34 N 0xd59ba4
0x14bfe1c T 0xdf00
0x4d9a2484 T 0xb33c
0x1b208 T 0x3394
0x64c5c N 0x061ac
0x6010 T 0xa0c38
0x796f34 T 0xd59ba4
0x16eb0 T 0x55a8fe3c
0x16eb0 N 0x55a8fe3c
0x4d9a2484 T 0xb33c
0x2116fc20 N 0x3efc
0x7464e8 T 0x90e2c
0x4f80370 N 0xf1d54
0x16eb0 T 0x55a8fe3c
0x6010 N 0xa0c38
0x055d4 N 0x3de04
0x0b8 T 0x7f38
0x16eb0 N 0x55a8fe3c
0xa94 N 0x3f0
0x36859bb4 N 0xd4c4
0x796f34 N 0xd59ba4
0x4f80370 T 0xf1d54
0x4f80370 N 0xf1d54
0xcb8 N 0xb529f5e4
0xa94 N 0x3f0
0xa0b14 T 0xbdc8930
0x6010 N 0xa0c38
0x1b208 T 0x3394
0xa0b14 T 0xbdc8930
0x36859bb4 N 0xd4c4
0x7464e8 T 0x90e2c
0xa0b14 N 0xbdc8930
0x0b8 N 0x7f38
0x796f34 T 0xd59ba4
0x4f80370 N 0xf1d54
0x796f34 N 0xd59ba4
0x4d9a2484 T 0xb33c
0x36859bb4 T 0xd4c4
0x16eb0 T 0x55a8fe3c
0x055d4 N 0x3de04
0x1b208 T 0x3394
0x64c5c N 0x061ac
0x1b208 N 0x3394
0x4d9a2484 T 0xb33c
0x0b8 N 0x7f38
0x2116fc20 T 0x3efc
0xa0b14 N 0xbdc8930
0x36859bb4 T 0xd4c4
0x2116fc20 N 0x3efc
0x1b208 T 0x3394
0x4f80370 N 0xf1d54
0xa94 T 0x3f0
0x1b208 T 0x3394
0x4f80370 N 0xf1d54
0x6010 N 0xa0c38
0xa00 N 0x9450165c
0xa0404cc N 0x53010350
0x6010 T 0xa0c38
0x4f80370 T 0xf1d54
0xa0b14 N 0xbdc8930
0x796f34 T 0xd59ba4
0x6010 T 0xa0c38
0xcb8 N 0xb529f5e4
0xcb8 T 0xb529f5e4
0x64c5c T 0x061ac
0xa0404cc N 0x53010350
0xa0b14 N 0xbdc8930
0x1b208 N 0x3394
0x7464e8 T 0x90e2c
0x36859bb4 T 0xd4c4
0x6010 N 0xa0c38
0x4f80370 T 0xf1d54
0xa94 T 0x3f0