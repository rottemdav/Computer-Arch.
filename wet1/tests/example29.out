0x6010 N 0x6014
0xa0404cc N 0xa0404d0
0xa00 N 0xa04
0x4f80370 N 0x4f80374
0xa0404cc T 0x53010350
0xa0404cc T 0x53010350
0x1b208 N 0x1b20c
0xb8 N 0xbc
0xa00 T 0x9450165c
0xa0b14 N 0xa0b18
0xcb8 N 0xcbc
0x6010 N 0x6014
0x796f34 N 0x796f38
0x14bfe1c N 0x14bfe20
0x4d9a2484 N 0x4d9a2488
0x1b208 T 0x3394
0x64c5c N 0x64c60
0x6010 N 0x6014
0x796f34 T 0xd59ba4
0x16eb0 N 0x16eb4
0x16eb0 N 0x16eb4
0x4d9a2484 T 0xb33c
0x2116fc20 N 0x2116fc24
0x7464e8 N 0x7464ec
0x4f80370 T 0xf1d54
0x16eb0 T 0x55a8fe3c
0x6010 N 0x6014
0x55d4 N 0x55d8
0xb8 N 0xbc
0x16eb0 T 0x55a8fe3c
0xa94 N 0xa98
0x36859bb4 N 0x36859bb8
0x796f34 N 0x796f38
0x4f80370 N 0x4f80374
0x4f80370 N 0x4f80374
0xcb8 N 0xcbc
0xa94 T 0x3f0
0xa0b14 N 0xa0b18
0x6010 T 0xa0c38
0x1b208 N 0x1b20c
0xa0b14 T 0xbdc8930
0x36859bb4 N 0x36859bb8
0x7464e8 T 0x90e2c
0xa0b14 T 0xbdc8930
0xb8 N 0xbc
0x796f34 N 0x796f38
0x4f80370 T 0xf1d54
0x796f34 N 0x796f38
0x4d9a2484 N 0x4d9a2488
0x36859bb4 N 0x36859bb8
0x16eb0 T 0x55a8fe3c
0x55d4 N 0x55d8
0x1b208 T 0x3394
0x64c5c T 0x61ac
0x1b208 T 0x3394
0x4d9a2484 T 0xb33c
0xb8 T 0x7f38
0x2116fc20 N 0x2116fc24
0xa0b14 T 0xbdc8930
0x36859bb4 T 0xd4c4
0x2116fc20 N 0x2116fc24
0x1b208 T 0x3394
0x4f80370 N 0x4f80374
0xa94 N 0xa98
0x1b208 N 0x1b20c
0x4f80370 T 0xf1d54
0x6010 T 0xa0c38
0xa00 T 0x9450165c
0xa0404cc N 0xa0404d0
0x6010 T 0xa0c38
0x4f80370 N 0x4f80374
0xa0b14 N 0xa0b18
0x796f34 N 0x796f38
0x6010 N 0x6014
0xcb8 N 0xcbc
0xcb8 T 0xb529f5e4
0x64c5c T 0x61ac
0xa0404cc T 0x53010350
0xa0b14 T 0xbdc8930
0x1b208 N 0x1b20c
0x7464e8 N 0x7464ec
0x36859bb4 N 0x36859bb8
0x6010 N 0x6014
0x4f80370 T 0xf1d54
0xa94 N 0xa98
flush_num: 42, br_num: 85, size: 1861b
